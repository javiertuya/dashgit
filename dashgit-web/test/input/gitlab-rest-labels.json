{
  "data": {
      "projects": {
          "nodes": [
              {
                  "id": "gid://gitlab/Project/117",
                  "name": "proj1",
                  "fullPath": "org/proj1",
                  "labels": {
                      "nodes": [ { "color": "#6699cc", "title": ".NET" }, { "color": "#CC0033", "title": "BLOCKING" } ]
                  }
              },
              {
                  "id": "gid://gitlab/Project/44",
                  "name": "projx",
                  "fullPath": "com/projx",
                  "labels": {
                      "nodes": [ { "color": "#CC0033", "title": "BLOCKING" }, { "color": "#00ffaa", "title": "Doing" } ]
                  }
              }
          ]
      }
  }
}