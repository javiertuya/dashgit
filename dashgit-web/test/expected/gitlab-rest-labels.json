{
  "org/proj1-.NET": {
    "color": "#6699cc"
  },
  "org/proj1-BLOCKING": {
    "color": "#CC0033"
  },
  "com/projx-BLOCKING": {
    "color": "#CC0033"
  },
  "com/projx-Doing": {
    "color": "#00ffaa"
  }
}